src/app/pages/Languages.svelte

```diff
@@ -52,12 +52,22 @@
   };
 
   /**
-   * Handles the click when the download language button is pressed. Should download a .kmp file.
-   * TODO: Replace stub
-   *
+   * Handles the click when the download language button is pressed. downloads a .kmp file.
    * @return {void}
    */
-  const handleDownload = (): void => {};
+  const handleDownload = async () => {
+    let { dictionaryName } = await worker.fetchAllCurrentProjectMetadata();
+
+    if (!dictionaryName) dictionaryName = "Example";
```

Comment:
Nit: What are your thoughts on making the default name "PredictiveTextStudioDictionary" or something of the sort?

Reply:
It's not a bad idea, I named it Example because that is what the one on the landing page is called. I can change them both to something more descriptive.

Reply:
On second thought, I will leave the landing page name as is because there is a-lot built around "Example.kmp". If the name has no significance we can change it in another PR.